examples/Classification/pointcloud_classification.py

```diff
@@ -74,24 +91,28 @@
 
     with torch.no_grad():
         for idx, batch in enumerate(tqdm(val_loader)):
-            pred = model(batch[0])
-            loss = criterion(pred, batch[1].view(-1))
+            category = batch['attributes']['category'].cuda()
+            pred = model(batch['data'].cuda())
+            loss = criterion(pred, category.view(-1))
             val_loss += loss.item()
 
             # Compute accuracy
             pred_label = torch.argmax(pred, dim=1)
-            val_accuracy += torch.mean((pred_label == batch[1].view(-1)).float()).cpu().item()
+            val_accuracy += torch.mean((pred_label == category.view(-1)).float()).cpu().item()
             num_batches += 1
 
     print('Val loss:', val_loss / num_batches)
```

Comment:
Add tiny `eps` to the denominator? (To prevent unforeseen errors, eg. if val batchsize is `0` by accident/intention)

Reply:
How can batchsize be 0 other than if the user purposely set it to be 0 ? If he does Dataloader would fail to construct `ValueError: batch_size should be a positive integer value, but got batch_size=0`

Reply:
I assume that you means `num_batches`, not `batchsize`. How can that happens in a way that requires us to support it ?

Reply:
Doesn't usually happen. I can see two cases in which this happens.
1. Missing data for particular category (could happen on CI, esp if we move to multiple machines)
2. User wilfully deletes data for particular category, to skip validation, as a hack

Again, I don't think this is super-important, as this isn't the intended use-case. I'll go by whatever you recommend.

Reply:
1) We definitely want it to crash ! Otherwise the missing data will go unnoticed.
2) We don't recommend people "hacking" our examples. (Neither training without cross-validation).